src/renderer/ui/components/modals/GoToModal.js

```diff
@@ -54,10 +81,18 @@ export default class GoToModal extends Component {
       remote.app.quit();
     } else if (url === 'DEBUG_INFO') {
       Emitter.fire('generateDebugInfo');
+      this.handleClose();
+    } else {
```

Comment:
We should maintain the pass-quick logic from before (If it matches the RegExp use the URL straight away otherwise run the resolver).